jerry-core/include/jerryscript-core.h

```diff
@@ -159,6 +159,18 @@ typedef struct
 } jerry_property_descriptor_t;
 
 /**
+ * Description of JerryScript heap memory stats.
+ * It is for memory profiling.
+ */
+typedef struct
+{
+  size_t version; /**< the version of the stats struct */
```

Comment:
@zherczeg I don't know how versioning of APIs with dynamic libraries this is usually dealt with, but I wonder whether it makes sense to have a over-arching "get version" API for version matching instead of adding versions for each and every API / ABI.